assets/src/components/editing/models/blockcode/Blockcode.scss

```diff
@@ -1,7 +1,6 @@
 @use 'delivery/variables' as delivery;
 
 .code-editor {
-  clear: both;
   margin-bottom: 10px;
```

Comment:
These clears used to be necessary when we were floating images left and right, but we don't support that any more